_site/assets/css/styles.css

```diff
@@ -0,0 +1,1115 @@
+body {
```

Comment:
Do not commit the `_site` directory. Make your changes in the `assets/css/styles.css` file.